### ISSUE TITLE
Chardet 3.0 causes backwards incompatible changes

### ISSUE THREAD

**Issue by balloob** [aio-libs/aiohttp#1810]
## Long story short
[Chardet 3.0 was released today](https://pypi.python.org/pypi/chardet). Aiohttp refers to chardet without a version boundary and so all new installs of aiohttp install Chardet 3.0. This is causing issues in aiohttp. This got caught by our test suite: https://travis-ci.org/home-assistant/home-assistant/jobs/221068366#L285

Aiohttp experienced similar problems with Yarl in the past. The solution for that was then to pin Yarl to a specific range.

## Expected behaviour

Short term: Chardet dependency to be limited to <3.

## Actual behaviour

Chardet 3.0 is getting installed.

## Steps to reproduce

```
git clone https://github.com/home-assistant/home-assistant.git
cd home-assistant
python3 -m venv .
source bin/activate
pip3 install tox
tox
```

## Your environment

Tests ran on Travis CI. Test failed on Python 3.4, 3.5, 3.6 and 3.6-dev.

## Workaround

Install `chardet==2.3` before installing aiohttp

**Comment by fafhrd91** [comment]
@asvetlov do you want to fix it or revert?

**Comment by seemethere** [comment]
sanic is also having issues with this as well. Issues are resolved if you also install `cchardet`

**Comment by asvetlov** [comment]
I'll cope with it tomorrow

**Comment by fafhrd91** [comment]
should be fixed by new version chardet

**Comment by asvetlov** [comment]
Just for the reference -- the issue is chardet/chardet#113
Fixed by chardet 3.0.2
